func_adl_xAOD/common/ast_to_cpp_translator.py

```diff
@@ -188,7 +188,6 @@ def get_rep(self, node: ast.AST, retain_scope: bool = False) -> crep.cpp_rep_bas
         # If it didn't work, this is an internal error. But make the error message a bit nicer.
         if not hasattr(node, 'rep'):
             raise Exception('Internal Error: attempted to get C++ representation for AST node "{0}", but failed.'.format(ast.dump(node)))
-
```

Comment:
It looks like all the changes in this file are deleting empty lines. Since this isn't changing the code, lets not do it (it makes this PR look bigger than it is). Could you make sure I am right, and if so, return the blank lines?

Reply:
I can do that!